### PR TITLE
[quant][graphmode][fx] Enable fuse handler for sequence of 3 ops

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69882
* #69778
* __->__ #69658

Summary:
This PR enables fuse handler for sequence of three ops, and merges all fuse handlers into one

TODO: we can also move this to backend_config_dict folder

Test Plan:
regression fusion test
```
python test/test_quantization.py TestFuseFx
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D32974907](https://our.internmc.facebook.com/intern/diff/D32974907)

### CODE REVIEW

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32974907).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32974907).

File: torch/ao/quantization/fx/fusion_patterns.py [hunk 1]
```diff
@@ -136,26 +72,41 @@ def fuse(self, quantizer: QuantizerCls,
...
         "undergoing changes, additoinal_fuser_method_mapping is not supported currently."
-        def get_module(n):
-            if n.op == "call_module":
-                return quantizer.modules[n.target]
-            elif n.op == "call_function" and n.target == torch.nn.functional.relu:
-                relu = torch.nn.ReLU()
-                relu.training = root_module.training
-                return relu
-            return MatchAllNode
+        def get_modules(pattern, modules):
```
Comment (vkuzo) [inline torch/ao/quantization/fx/fusion_patterns.py:75]: can we add a quick docblock with an example of what this is doing?

**Review by vkuzo — APPROVED** [review]

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32974907).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32974907).

**Pull request closed without merging**